lib/src/over_react_redux/over_react_flux.dart

```diff
@@ -0,0 +1,599 @@
+          ...mapActionsToPropsWithOwnProps(
+              actionsForStore[state] as TActions, ownProps),
+        };
+      };
+
+      mapStateToProps = null;
+    }
+
+    // Special case: the converse of case7, set mapStateToPropsWithOwnProps and
+    // include mapActionsToProps because the actions still need to be set and
+    // cannot be set on mapStateToProps.
+    if (case8) {
+      final originalMapStateWithOwnProps = mapStateToPropsWithOwnProps;
+      Map wrappedMapStateToPropsWithOwnProps(TStore state, TProps ownProps) {
+        return {
+          ...originalMapStateWithOwnProps(state, ownProps),
+          ...mapActionsToProps(actionsForStore[state] as TActions),
+        };
+      }
+
+      mapStateToPropsWithOwnProps = wrappedMapStateToPropsWithOwnProps;
+    }
+  }
+  /*--end usage of cases--*/
+
+  if (areStatePropsEqual == null) {
+    const defaultAreStatePropsEqual = _shallowMapEquality;
+    const propHasher = CollectionLengthHasher();
+    bool areStatePropsEqualWrapper(TProps nextProps, TProps prevProps) {
+      final result = defaultAreStatePropsEqual(nextProps, prevProps);
+
+      assert(() {
+        prevProps.forEach((key, value) {
+          // If the value is the same instance, check if the instance has been mutated,
+          // causing its hash to be updated
+          if (identical(value, nextProps[key]) &&
+              propHasher.hasHashChanged(value)) {
+            window.console.warn(
+                'connect: The instance of the value mapped from store "$TStore" to prop "$key" was mutated directly, which will prevent updates from being detected.'
+                ' Instead of mutating datastructure instances within the store, overwrite them with modified copies.\n'
+                '\n  Good: `_items = [..._items, newItem]`'
+                '\n  Bad:  `_items.add(newItem)`'
+                '\n\nIf this is not possible, then either:'
+                '\n - update `areStatePropsEqual` to reflect this behavior: `areStatePropsEqual: (_, __) => false`'
+                '\n - set `pure: false`');
+          }
+        });
+
+        return true;
+      }());
+
+      return result;
+    }
+
+    areStatePropsEqual = areStatePropsEqualWrapper;
+  }
+
+  return connect(
+    mapStateToProps: mapStateToProps,
+    mapStateToPropsWithOwnProps: mapStateToPropsWithOwnProps,
+    mergeProps: mergeProps,
+    areOwnPropsEqual: areOwnPropsEqual,
+    areStatePropsEqual: areStatePropsEqual,
+    areMergedPropsEqual: areMergedPropsEqual,
+    context: context,
+    pure: pure,
+    forwardRef: forwardRef,
+    // Don't pass along the dispatcher as props by default
+    mapDispatchToProps: (_) => const {},
+    // This will always be false, since both arguments will also be the same store
+    areStatesEqual: (_, __) => false,
+  );
+}
+
+extension InfluxStoreExtension<S extends InfluxStoreMixin> on S {
+  /// Returns a [FluxToReduxAdapterStore] instance from the Flux store instance.
```

Comment:
#nit `FluxToReduxAdapterStore` is redundant here https://dart.dev/guides/language/effective-dart/documentation#avoid-redundancy-with-the-surrounding-context

#nit prefer "this" or "this store" over "the store": https://dart.dev/guides/language/effective-dart/documentation#prefer-using-this-instead-of-the-to-refer-to-a-members-instance

non-#nit: Having something mentioning that this is idempotent would be helpful:
```suggestion
  /// Returns an adapter store backed by this store, which can be used with [connect].
  /// 
  /// Multiple calls to this method will always return the same instance.
```